red/ethernet-ip.js

```diff
@@ -282,6 +291,8 @@ module.exports = function (RED) {
 
             connected = false;
             node._plc = new Controller();
+            node._plc.removeAllListeners("close");
```

Comment:
Do you know if this call to "removeAllListeners" is really needed? I've once read that this is often a bad practice, because it may break other (sometimes internal) event listeners.

Reply:
I didn't see any other listeners attached to that event in the ethernet-ip library, but I am going to tweak this approach a bit to make it more focused and update the PR.